neo4j-browser-5.26.yaml

```
@@ -0,0 +1,76 @@
+package:
+  name: neo4j-browser-5.26
```

We'll just pin to the version in neo4j

```suggestion
  name: neo4j-browser
```